app/templates/event/share_event.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet"
+          integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+</head>
+<body>
+    <div class="share_event card"  style="width: 30rem;">
+            <h5>Hi!</h5>
```

Comment:
Why `h5`?

Reply:
that the format from bootstrap, and it looks nice, you don't think?

Reply:
h5 is a title element :) I'm sure you did great work regarding the design, but you should not use `h5` for design purposes. Please use CSS to get the same effect.